src/main/java/com/example/producerconsumer/QueueConsumerImpl.java

```diff
@@ -25,11 +25,16 @@ public QueueConsumerImpl(String name, List<String> messagesOutputList, QueueElem
     }
     
     @Override
+    //kolejność metod powinna być odwrotna
```

Comment:
dlaczego ?

Reply:
Methods in a class should be organized per level of abstraction, where the methods used by a method sit directly below it
https://herbertograca.com/2016/09/03/clean-code-3-functions-by-robert-c-martin-uncle-bob/

Polecam tę pozycję - jeśli nie miał Pan jej w rękach to zapewniam, że warto - mój faworyt. Czytałem kilka lat temu i wlasnie planuje do niej wrócić ponownie.
https://helion.pl/ksiazki/czysty-kod-podrecznik-dobrego-programisty-robert-c-martin,czykov.htm